spinkube/install.sh

```diff
@@ -1,7 +1,24 @@
 #!/bin/sh
+#Check we are on Talos
+if $(kubectl get nodes -o json | jq .items.[].status.nodeInfo.osImage | grep -q "Talos"); then 
+    echo "Found Talos as part of OS Image string"; else
+    echo "Did not find Talos as part of the OS Image string. Exiting as only Talos is supported" ;
+    exit
+fi
+
 #Cert manager check 
+#If namespace, or deployment is not found, the wait command below will fail, and the script continues. 
+until $(kubectl get namespaces cert-manager -o name --ignore-not-found | grep -q "namespace/cert-manager"); do 
+    echo "Waiting for cert-manager namespace creation"; 
+    sleep 1;
+done
+
+until $(kubectl get deployment cert-manager-webhook -n cert-manager --ignore-not-found -o name | grep -q "deployment.apps/cert-manager-webhook"); do
+    echo "Waiting for cert-manager-webhook deployment";
+    sleep 1;
+done
 
-kubectl wait --for=condition=available --timeout=300s deployment/cert-manager-webhook  -n cert-manager
+kubectl wait --for=condition=available --timeout=300s deployment/cert-manager-webhook  -n cert-manager 
 
 kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.1.0/spin-operator.runtime-class.yaml
```

Comment:
If version is changign then this has to be updated as well .

Reply:
@mikkelhegn ^^

Reply:
Sorry - I meant the version of the app in the marketplace in spinkube/manifest.yaml

Reply:
IF we change that to 0.2 then shouldn't it be changed at install level as well ?

Reply:
Spinkube is still v0.1.0.

My intention was that the app (install method etc.) change enough to bump the installer (marketplace app) version. Do you want to have the marketplace app version in `manifest.yaml` to only change when SpinKube changes the version?

Reply:
Yeah usually the install version is the release version that we tie to

Reply:
I've reverted the version change